rules/no-watch/no-watch.ts.test.js

```diff
@@ -0,0 +1,105 @@
+const { RuleTester } =
+  require("@typescript-eslint/experimental-utils").ESLintUtils;
+const { join } = require("path");
+
+const { readExample } = require("../../utils/read-example");
+const rule = require("./no-watch");
+
+const ruleTester = new RuleTester({
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+    project: "./tsconfig.json",
+    tsconfigRootDir: join(__dirname, ".."),
+  },
+});
+
+const readExampleForTheRule = (name) => ({
+  code: readExample(__dirname, name),
+  filename: join(__dirname, "examples", name),
+});
+
+ruleTester.run("effector/no-watch.ts.test", rule, {
+  valid: ["correct.ts"].map(readExampleForTheRule),
+
+  invalid: [
+    ...["incorrect-with-effect.ts"]
+      .map(readExampleForTheRule)
+      .map((result) => ({
+        ...result,
+        errors: [
```

Comment:
This pattern leads to hard debugging. Could you split incorrect test-cases and put it by files with only one error?

Reply:
Do you mean that for every incorrect case there should be a separate file?
1 error per file

For example: effects has 5 cases => 5 files

Am I right?

Reply:
Hm, in this case up to you. I think, it's easier to debug in case of broken tests, but I can be wrong here.

Reply:
You right. I choose a solution in the middle. No more than 2 errors per case.